### PR TITLE
String n feature

### PR DESCRIPTION
**Pull request by hiral34**
Hello Sir,
As per instructions, I have created an empty project, then in new branch done my regex homework as well as a small feature file and now asking for merge request. Please let me know the outcome.

### CODE REVIEW

File: src/Operators.java [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+    }
+    public static void replace(String str1, String str2, String str3){
+        System.out.println(str1.replaceAll(str2,str3));
+    }
+    public static void replacechars(String str1, String str3){
+        str1 = str1.replaceAll("\\.+","");
+        str1 = str1.replaceAll(" ","");
+        System.out.println(str1.replaceAll("\\w+",str3));
+    }
+    public static void middlechar(String str1){
```
Comment (rdankhara) [inline src/Operators.java]: good method names should be camelCasing e.g. middleChar

File: src/Operators.java [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
-
-    public static void main (String[] args){
-
+    public static void length(String str1){
+        System.out.println("The length of the string '" + str1 + "' is: " + str1.length());
+    }
+    public static void replace(String str1, String str2, String str3){
+        System.out.println(str1.replaceAll(str2,str3));
+    }
+    public static void replacechars(String str1, String str3){
```
Comment (rdankhara) [inline src/Operators.java]: This will replace spaces and . + symbols as well along with another string

File: src/Login.feature [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Feature: Login Successful
+  Scenario: User should be able to login successfully with valid username and password.
+    Given User is on Home Page of given website
+    When user clicks on Sign In Page
+    And enters Valid Credentials
+    Then he should be able to see the welcome page
```
Comment (rdankhara) [inline src/Login.feature]: I think you need to cover more scenarios min 7 e.g 
Login with invalid username and valid password 
Login with valid username and invalid password 
Login with empty username and valid password etc.

**Review by rdankhara — CHANGES_REQUESTED** [review]
Please apply changes, Login.feature has only one scenario, where as asked atleast 7

File: src/Login.feature (hunk 3 above)
Comment (hiral34) [reply src/Login.feature]: I have added 7 more scenarios to the original one.

File: src/Operators.java (hunk 2 above)
Comment (hiral34) [reply src/Operators.java]: Is it not suppose to replace space, . and + sign?
I have now sorted this issue, so the method will now not replace . and + sign but I think spaces should still be removed. Please let me know if not, I will amend it.
Thanks,
Hiral

**Review by rdankhara — APPROVED** [review]
LGTM (Looks good to me), you can merge it on git hub by clicking merge button,  but then to take that on your computer your will need to switch to master branch using git checkout master and then git fetch and then git pull command should update your local master branch

**Pull request merged**